common/src/limits.rs

```diff
@@ -10,32 +10,44 @@ pub struct Limits {
     /// The amount of projects this user can create.
     project_limit: u32,
     /// Whether this user has permission to provision RDS instances.
+    #[deprecated(
+        since = "0.38.0",
```

Comment:
Nice! Should it be 0.37? The current version is 0.36.
```suggestion
        since = "0.37.0",
```

Reply:
I assumed this PR will only make it into the next release (if we release today) :man_shrugging:

Reply:
We can update this if it is released earlier :smile: